src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -32,21 +32,28 @@
 #  This module provides tools to handle point snapping and
 #  everything that goes with it (toolbar buttons, cursor icons, etc.).
 
+import FreeCAD as App
```

Comment:
Please notice that we want to import first the generic modules, that is, modules provided by Python itself, or Python packages; at the end we want to import our own modules, meaning `FreeCAD` et al.

This is the reason I placed the import of FreeCAD after the Qt imports. Qt imports are more generic, while FreeCAD, Draft, DraftVecUtil, etc., are more specific.

Reply:
ok, i thought FreeCAD had to be the first one... no problem to change this.